src/js/resize.js

```diff
@@ -0,0 +1,31 @@
+import Split from 'split.js'
+
+const PAGE_WIDTH = 960
+
+const main = document.getElementById('main')
```

Comment:
As I mentioned previously you don't really need to store reference to main as you can access it through `gutter.parentNode` or something along these lines :)